ocsp/responder/multi_source_test.go

```diff
@@ -13,38 +13,122 @@ import (
 	"golang.org/x/crypto/ocsp"
 )
 
-type succeedSource struct {
-	resp *Response
+type ok struct{}
+
+const expectedFreshness = 60 * time.Hour
```

Comment:
nit: this line is out of order, appearing between `type ok` and `func (ok) Response(...)`.